How-to-upgrade-review-version.md

```diff
@@ -0,0 +1,399 @@
+# How to Upgrade to TypeScript Without Anybody Noticing
+
+This guide will show you how to upgrade to Typescript without anybody
+noticing. Well, people *might* notice &mdash; what I really mean is
+that you won't have to change your build at all. You'll have the
+ability to get errors and completions in your editor and run `tsc`
+from the command line, but you won't have to integrate Typescript into
+your build.
+
+The way to do this is to not use Typescript. Stick with Javascript and
+use JSDoc to provide type information. This is less convenient than
+Typescript's syntax for types, but it means that your files stay
+boring old Javascript.
+
+Here's how. I'm going to use
+[typescript-eslint-parser](https://github.com/eslint/typescript-eslint-parser)
+as an example package so you can follow along if you want. Notably,
```

Comment:
Since your code examples are so specific, maybe link to the specific commit you used because they may stop making sense if the package changes in the future